test/test_loader.py

```diff
@@ -178,7 +178,7 @@ def test_load_all_other_type(self):
             self.loader.load_all_msgs()
             self.mock_db.load_records.assert_called_once()
             mock_log.assert_has_calls(
-                        [call('Message contains %i records', 1)])
+                        [call('Message contains 1 %s record', 'Storage')])
```

Comment:
```suggestion
                        [call('Message contains 1 Storage record')])
```
why do we need the %s? Why not just this?

Reply:
Ah right. So, We are just mocking and expecting that it would be called something like this `[call('Message contains 1 %s record', 'Storage')])` and we are not checking how it would be evaluated @rowan04 . As you suggested I agree that end result we would be getting is as you suggested. Since we are not checking the actual log. We just expect it would be calling something like this ` [call('Message contains 1 %s record', 'Storage')])`. Does that make sense ?

Reply:
yeah my bad, i see now